src/components/users.js

```diff
@@ -169,13 +197,35 @@ export const UserList = props => {
         />
         <TextField source="id" sortBy="name" />
         <TextField source="displayname" />
-        <BooleanField source="is_guest" />
-        <BooleanField source="admin" />
         <BooleanField source="deactivated" />
+        {adminFilter && <BooleanField source="admin" />}
+        {isGuestFilter && <BooleanField source="is_guest" />}
+        {creationTimeFilter && (
```

Comment:
The columns should always be visible.

Reply:
Than the list is pretty wide if you do not have short user names.